extension/Config.jsm

```diff
@@ -18,7 +18,11 @@ const config = {
       * - If there is no key for an endStudy reason, no url will open.
       * - usually surveys, orientations, explanations
       */
-    endings: {},
+    endings: {
+      expired: {
+        baseUrl: "https://qsurvey.mozilla.com/s3/sharing-study",
```

Comment:
I think this survey should be shown also for `reason=user-disable`, since that is the reason that will happen if Shield uninstalls this add-on because the study has ended. Maybe check with Kamyar though on this.

Reply:
Kamyar and matt_g confirm that we should also show for `reason=user-disable`. The SurveyGizmo URL is passed some arguments which include the reason and can adjust the survey content as needed.